packages/idx/src/property/search.component.ts

```diff
@@ -17,6 +17,8 @@ import '@stratusjs/idx/idx'
 // Stratus Dependencies
 import {isJSON} from '@stratusjs/core/misc'
 import {cookie} from '@stratusjs/core/environment'
+// FIXME should we be renaming the old 'stratus.directives' variables to something else now that we're @stratusjs?
```

Comment:
At the moment, the Stratus Runtime still looks up the directives by that naming scheme in the config.  As soon as we have a replacement for that, we can begin using normal SystemJS names.